code/src/components/TaskList.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import moment from 'moment/moment';
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { task } from 'reducers/task';
+
+export const TaskList = () => {
+  const tasklist = useSelector((store) => store.tasks.items);
+  const dispatch = useDispatch();
+
+  const onDeleteAllTasksClick = (id) => {
+    dispatch(task.actions.deleteSingleTask(id))
+  }
```

Comment:
Maybe this function could have a more precise name as it's actually not deleting all tasks but rather a single task?